markets/perps-market/contracts/modules/AsyncOrderModule.sol

```diff
@@ -108,14 +104,42 @@ contract AsyncOrderModule is IAsyncOrderModule {
         uint128 marketId,
         int128 sizeDelta
     ) external view override returns (uint256 orderFees, uint256 fillPrice) {
-        (orderFees, fillPrice) = _computeOrderFees(marketId, sizeDelta);
+        (orderFees, fillPrice) = _computeOrderFees(
+            marketId,
+            sizeDelta,
+            PerpsPrice.getCurrentPrice(marketId, PerpsPrice.Tolerance.DEFAULT)
+        );
     }
```

Comment:
I think it would make sense to create a `withPrice` version of this function as well.